geo-types/src/line_string.rs

```diff
@@ -149,6 +149,21 @@ impl<T: CoordinateType> LineString<T> {
             }
         }
     }
+
+    /// Return the number of coordinates in the `LineString`.
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// use geo_types::LineString;
+    ///
+    /// let mut coords = vec![(0., 0.), (5., 0.), (7., 9.)];
+    /// let line_string: LineString<f32> = coords.into_iter().collect();
+    /// assert_eq!(3, line_string.len());
+    /// ```
+    pub fn len(&self) -> usize {
```

Comment:
hum I'm not sure about this name, wouldn't `nb_points` be more explicit ? The length of a linestring might mean different things.

Reply:
True, I will rename it to `nb_coords`. Thanks!